OpenXmlPowerTools/WmlToHtmlConverter.cs

```diff
@@ -439,7 +509,17 @@ private static void SetStyleElementValue(XElement xhtml, string styleValue)
             // Transform contents of runs.
             if (element.Name == W.r)
             {
-                return ConvertRun(wordDoc, settings, element);
+                //return ConvertRun(wordDoc, settings, element);
```

Comment:
This is where the PR falls over. There's an earlier PR that changes this particular bit of code.